youtube_dl/extractor/youtube.py

```diff
@@ -516,6 +516,24 @@ class YoutubeIE(YoutubeBaseInfoExtractor):
                 'skip_download': 'requires avconv',
             }
         },
+        # DASH manifest with segment_list
+        {
+            'url': 'https://www.youtube.com/embed/CsmdDsKjzN8',
+            'md5': '8ce563a1d667b599d21064e982ab9e31',
+            'info_dict': {
+                'id': 'CsmdDsKjzN8',
+                'ext': 'mp4',
+                'upload_date': '20150510',
```

Comment:
It's quite weird, while in the youtube webpage it says it was uploaded `May 10, 2015`,  the first element where we try to get the date says:
```
<meta itemprop="datePublished" content="2015-05-01">
```

I will change it to `20150501` for now.